src/components/do-dont/DoDont.tsx

```diff
@@ -0,0 +1,25 @@
+import { GoAIcon } from "@abgov/react-components";
+import "./DoDont.css";
+import { ReactNode } from "react";
+
+type DoDont = "do" | "dont";
+interface Props {
+    type: DoDont;
+    content: ReactNode;
```

Comment:
Can you change the content to be `children`. This will make it cleaner when passing html to the component.